testing/endtoend/e2e_test.go

```diff
@@ -280,6 +272,7 @@ func runEndToEndTest(t *testing.T, cfg *e2eConfig) {
 		baseChallengeManagerOpts,
 		challengemanager.WithAddress(txOpts.From),
 		challengemanager.WithName(name),
+		challengemanager.WithAPIEnabled("localhost:7257", "/tmp/mydb"),
```

Comment:
Does anything clean up the "/tmp/mydb" path?
You could use https://pkg.go.dev/testing#T.TempDir to put it in a directory that would automatically be removed at the end of the test.